entrypoint.sh

```diff
@@ -31,24 +31,24 @@ if [[ $INPUT_RUBOCOP_VERSION = "gemfile" ]]; then
     RUBOCOP_VERSION=$INPUT_RUBOCOP_VERSION
 fi
 
-gem install -N rubocop $(version $RUBOCOP_VERSION)
+gem install -N rubocop --version "${RUBOCOP_VERSION}"
```

Comment:
If this action is performed without the `rubocop_version` input, we will get the error:
```bash
$ gem install -N rubocop --version
ERROR:  While executing gem ... (OptionParser::MissingArgument)
    missing argument: --version
```

Reply:
It actually doesn't fail in the script.  If `INPUT_RUBOCOP_VERSION` is not defined, the call ends up being

`gem install -N rubocop --version ""`

Note the empty sting.  gem handles this and just installs the latest version.  You are right though if the quotes weren't there as `gem install -N rubocop --version` fails as you indicate.

Reply:
Thank you for explaining me 👍